Source/SwiftLintFramework/Extensions/String+md5.swift

```diff
@@ -0,0 +1,20 @@
+#if canImport(CommonCrypto)
+import CommonCrypto
+#else
+import CryptoSwift
+#endif
+import Foundation
+
+#if canImport(CommonCrypto)
+extension String {
```

Comment:
Aren't all these other imports unused?

```suggestion
#if canImport(CommonCrypto)
import CommonCrypto

extension String {
```

Reply:
I thought `CryptoSwift` is just fallback, if `CommonCrypto` is not available.

Reply:
Correct, but CryptoSwift already comes with `String.md5()`, so what this code does is add the same convenience method to CommonCrypto.

Reply:
I just copied the code from `Source/SwiftLintFramework/Extensions/Configuration+Cache.swift`, I can't decide if the code could be throw away.

Reply:
That file uses CryptoSwift but this file doesn’t.

Reply:
I suggest to split this in an own issue/pr.